doc/decisions/decision_process.md

```diff
@@ -24,13 +26,16 @@ Substantial decisions must be made in a transparent and participative way.
   - Rebase only if the decision was already accepted and has a merge conflict.
 - For reviewers:
   - Prefer to directly give suggestions how to change sentences.
-  - General questions should be asked in the root of "Conversation" and not at random sentences in the review.
+  - General questions should be asked in the root of "Conversation" and not at vaguely related sentences in the review.
+- Decision PRs do not significantly change anything but one decision to make discussions clearly structured.
```

Comment:
Funny how this very PR here is the opposite of this sentence. 😄

Reply:
Maybe the sentence was misunderstood, I removed to wrong rationale: bdb29bab5140c8ad406acdf1465f82b5776ab354
But yes, adding the man_pages here was definitely not a good idea.